contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -86,16 +91,22 @@ contract TrueCreditAgency is UpgradeableClaimable {
         creditAdjustmentCoefficient = 1000;
         utilizationAdjustmentCoefficient = 50;
         utilizationAdjustmentPower = 2;
+        repaymentPeriod = 31 days;
+    }
+
+    modifier onlyAllowedBorrowers() {
```

Comment:
When I tried to auto-resolve merge conflicts with the base branch, Github duplicated this function. We probably want to remove this modifier change from this branch.